packages/common-utils/src/metadata.ts

```diff
@@ -13,7 +13,7 @@ import type { ChartConfigWithDateRange } from '@/types';
 
 const DEFAULT_SAMPLE_SIZE = 1e6;
 
-class MetadataCache {
```

Comment:
extra export?

Reply:
not right now. in case later we move the initialization code to the client side